src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,220 @@
+trigger:
+- master
+- main
+- release
+
+# trigger this pipeline if there's a PR to any of these branches
+pr:
+- master
+- main
+- staging
+- releases/*
+
+variables:
+- name: jobTimeout
+  value: '60'
+
+stages:
+  - stage: Build_UNO
+
+    jobs:
+#if (wasm)    
+    - job: Wasm
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'wasm'
+
+        - template: /azure/steps/wasm.yml
+          parameters:
+            package_id: 'Wasm'
+            project_to_build: 'MyExtensionsApp.Wasm/MyExtensionsApp.Wasm.csproj'
+      pool:
+        vmImage: macOS-latest   
+#endif
+
+#if (android) 
+    - job: Android
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'android'
+
+        - template: /azure/steps/android.yml
+          parameters:
+            package_id: 'Android'
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+      pool:
+        vmImage: macOS-latest
+
+    ###################################################
+    #                    IMPORTANT                    #
+    # Store certificates and mobile provision files   #
+    # in secure files, then just replace the jks      #
+    # variable down below                             #
+    # with the appropiate filename                    #
+    ###################################################
+
+    - job: Android_Signed   
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'android'
+        - template: /azure/steps/android.yml
+          parameters:
+            sign_package: 'true'
+            certificate:  '' #YOUR_CERT.jks stored in secureFiles
+            certificate_password: 'contoso'
+            certificate_alias: 'contoso'
+            package_id: 'signed_android'
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+      pool:
+        vmImage: macOS-latest         
+#endif
+
+#if (ios) 
+    - job: iOS
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            target_platform: 'ios'
+
+        - template: /azure/steps/mobile.yml
+          parameters:
+            project_to_build: 'MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj'
+            target_framework: 'net$ciTargetDotNetVersion$.0-ios'
```

Comment:
Do you need to add target_framework to the other jobs that use the mobile.yml steps (eg Android, the iOS_signed)